tool/main.re

```diff
@@ -247,14 +247,12 @@ static MULTIPLAYER_MENU = Ui::new("Multiplayer:", List::of(
     }),
 ));
 
-static mut SAVE_RECORDING_LABEL = Text { text: "Save Recording" };
 static mut RECORDING_NAME_LABEL = Text { text: "Recording name" };
-static mut LIST_RECORDINGS_LABEL = Text { text: "List Recordings" };
-static mut DELETE_RECORDING_LABEL = Text { text: "Delete Recording" };
 
-fn main_list_recordings() {
-    let recordings_list = Tas::list_recordings();
-    LIST_RECORDINGS_LABEL.text = f"List Recordings ({recordings_list})";
+fn triple_leave_ui() {
+    leave_ui();
+    leave_ui();
+    leave_ui();
 }
```

Comment:
This is not what I meant with https://github.com/oberien/refunct-tas/pull/85#discussion_r853996914. As you can see, you needed to call this function from three different places. In all of those three places you needed to change the code to have the `Back` button as first element of the list. Try refactoring all of the triplicated code into one function, such that a change like moving the `Back` button will only need to be changed in a single place.